src/utils/trust.involved.ts

```diff
@@ -187,10 +225,28 @@ const getUrl = (isUpdate: boolean) => {
   }
 };
 
-const getNextPage = (isUpdate: boolean) => {
+const getNextPage = (isUpdate: boolean, isReview: boolean) => {
+  if (isReview) {
+    return config.UPDATE_MANAGE_TRUSTS_ORCHESTRATOR_URL;
+  }
   if (isUpdate) {
     return config.UPDATE_TRUSTS_ASSOCIATED_WITH_THE_OVERSEAS_ENTITY_URL;
   } else {
     return `${config.TRUST_ENTRY_URL + config.ADD_TRUST_URL}`;
   }
 };
+
+const moveTrustOutOfReview = (appData: ApplicationData, trustId: string) => {
```

Comment:
I think this maybe should go in **utils/update/review_trusts**. 

Though following on from the comment above, just the first line would need to change, to get the index using the `in_review` flag:

```
const trustIndex = (appData.update?.review_trusts ?? []).findIndex(trust => trust?.review_status?.in_review);
```

Reply:
Moved